File: java/p001-p009/Problem 1 - Multiples of 3 and 5/Problem1.java
public class Problem1 {

	public static void main(String[] args) {
		// Creates a variable to store the sum value
		int multiples = 0;

		/* For loop to calculate the multiples
		   and add them to the final variable */
		for(int i=1; i<1000; i++) {
			/* Condition which checks if the
			   number is a multiple of 3 or 5 */
			if((i % 3 == 0) || (i % 5 == 0))
				multiples += i; // Adds the respective number to the final variable
		}
		System.out.println(multiples); // Prints out the final value

	}

}

File: c/p001-p009/Problem 1 - Multiples of 3 and 5/Problem1.c
#include <stdio.h>

// Main method
int main(int argc, char const *argv[]) {
    // Declaring and Assigning the variable
    int multiples;
    multiples = 0;

    /* For Loop to go through the numbers and
       add the numbers which are factors of 3 or 5 */
    for (int i = 1; i < 1000; i++) {
        // Condition to check if the number is a factor of 3 or 5
        if ((i % 3 == 0) || (i % 5 == 0)) {
            // Adds the number to the variable
            multiples += i;
        }
    }

    // Prints out the result
    printf("%d \n", multiples);
}
